lib/esyBuildEjectCommand.js

```diff
@@ -239,8 +240,10 @@ function buildEjectCommand(
           type: 'rule',
           target: packageRoot,
           command: withPackageEnv(outdent`
-            rm -rf $cur__root; \\
-            cp -r $cur__source_root $cur__root
+            if [ ! -d "$cur__install" ]; then \\
+              rm -rf $cur__root; \\
+              rsync --quiet --archive $cur__source_root/ $cur__root --exclude $cur__root; \\
```

Comment:
We esclude `$cur__root` here because in case of the root package — we don't rsync `_build` into itself.